typescript/infra/src/govern/HyperlaneAppGovernor.ts

```diff
@@ -346,7 +347,11 @@ export abstract class HyperlaneAppGovernor<
         // Will throw if the transaction fails
         await multiProvider.estimateGas(chain, call, submitterAddress);
         return true;
-      } catch (e) {} // eslint-disable-line no-empty
+      } catch (e) {
```

Comment:
Helped with debugging

Reply:
might be a little noisy / concerning to someone less familiar with the implementation and just running tooling, as we expect this to happen a lot as part of deciding which submission type works. Maybe `rootLogger.debug` instead of `console.error` could help? and/or the message suggesting that the error is possibly expected

Reply:
Fair